cmd/ci-secret-bootstrap/main.go

```diff
@@ -68,16 +68,26 @@ func (o *options) validateOptions() error {
 	return nil
 }
 
+var (
+	secrets = sets.NewString()
```

Comment:
We should not use global variables. If you're threading through a closure to the client anyway, just update a local variable in `main()`.

Reply:
Updated.